### ISSUE TITLE
Sort order in Quick Filter is different from the rest of the app

### ISSUE THREAD

**Issue by djibux** [mpcjanssen/simpletask-android#719]
The Quick Filter lists elements in a different order than the list selection popup.

I will give an example.  Let's say I have lists starting with a `!`, lists that are purely alphabetical, and lists starting with a `~`.  I would expect them to be sorted as follows, in the ASCII table order (and indeed it is the way they are sorted in the list selection popup).
```
  @!Important
  @Home
  @Work
  @~Later
```

In the quick filter, and in fact the main view, items starting with a `~` are sorted before alphabetical elements.
```
  @!Important
  @~Later
  @Home
  @Work
```

I don't really care what order is used, but I believe it should be consistent within the app.

**Comment by smichel17** [comment]
Pretty sure this is a dupe of #239

**Comment by djibux** [comment]
I think you're right.  Sorry, I did do a search but missed it somehow.

**Comment by smichel17** [comment]
No worries. I probably wouldn't have found it if I didn't know it existed already. When I searched for "sort" it didn't show up, only "sort case" showed it. Go figure.

**Comment by mpcjanssen** [comment]
Fixed in #239.
